lib/Doctrine/Migrations/Metadata/AvailableMigration.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Migrations\Metadata;
+
+use Doctrine\Migrations\AbstractMigration;
+use Doctrine\Migrations\Version\Version;
+
+class AvailableMigration
+{
+    /** @var Version */
+    private $version;
+
+    /** @var AbstractMigration */
+    private $migration;
+
+    public function __construct(Version $version, AbstractMigration $migration)
```

Comment:
I am for making the ctor private and a have a static named constructor to be immutable.

Also I am an for version() and migration() without the get prefix